ci/container_scripts/build_and_install.sh

```diff
@@ -34,5 +34,5 @@ case "$BUILDTYPE" in
         ;;
 esac
 
-./setup build -j4 --test --extra --werror $OPTIONS
-./setup install
+./setup build -j4 --test --extra --werror --prefix=/opt/shadow $OPTIONS
+./setup install
```

Comment:
Why change the installation directory? I think it would be better to run tests using the default options here.

Reply:
Installing into a dedicated directory `/opt/shadow` instead of merging into e.g. ~/.local with other things allows us to easily copy out a "releasable" shadow as in https://github.com/shadow/shadow/issues/2389#issuecomment-1232264491. It is a bit orthogonal from the meat of this PR though. I'll leave it out for now.